riotfile.py

```diff
@@ -1048,6 +1048,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="mysql",
             command="pytest {cmdargs} tests/contrib/mysql",
+            env={"DD_DBM_PROPAGATION_MODE": "full"},
```

Comment:
```suggestion
```

Reply:
Flaws in our dbm procedure support were detected by running mysql tests with `DD_DBM_PROPAGATION_MODE=full`. 

I am tempted to keep this environment variable in this PR because it can serve as a regression test.

Reply:
I don't see an issue with keeping it in but maybe we should comment it?